<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>857</width>
    <height>569</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>90</y>
      <width>541</width>
      <height>341</height>
     </rect>
    </property>
    <row>
     <property name="text">
      <string>1</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>2</string>
     </property>
    </row>
    <column>
     <property name="text">
      <string>1</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>2</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="ButtonAddRow">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>110</y>
      <width>80</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>AddRow</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ButtonDelRow">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>190</y>
      <width>80</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>DelRow</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBoxRow">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>110</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="ButtonGetSum">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>230</y>
      <width>80</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>GetSum</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelSum">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>190</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="ButtonAddColumn">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>70</y>
      <width>80</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>AddColumn</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBoxColumn">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>70</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="ButtonDelColumn">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>150</y>
      <width>80</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>DelColumn</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ButtonRandom">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>270</y>
      <width>80</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Random</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>857</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
